packages/ruby-zpng/PKGBUILD

```
@@ -0,0 +1,27 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+_gemname=zpng
```

```suggestion
_gemname=${pkgname#ruby-}
```

and put after pkgname